rmw_cyclonedds_cpp/include/rmw_cyclonedds_cpp/TypeSupport.hpp

```diff
@@ -78,7 +78,7 @@ struct StringHelper<rosidl_typesupport_introspection_c__MessageMembers>
     return std::string(data.data);
   }
 
-  static void assign(cycdeser & deser, void * field, bool)
+  static void assign(cycdeser & deser, void * field)
```

Comment:
don't you need to do the same in L101?

Reply:
Yes, we need to remove it. Sorry for my mistake.
I'll retest them(including rcl, rclcpp, system_tests)